### PR TITLE
Update pubs list

### PR DESCRIPTION
**Pull request by jen-martin**
Addresses #400. Updates publications from Melissa, Chris and Peter's pubs on pubmed for 2019-2021.

### CODE REVIEW

**Review by jmcmurry — APPROVED** [review]
There are a couple probably missing (eg. LIRICAL / LOINC2HPO), but that shouldn't hold this batch up. Please merge.

**Review by monicacecilia — CHANGES_REQUESTED** [review]
Thank you for working on this, Jen. I've added a few comments for removal of some of these entries.

File: src/views/Publications.md [hunk 1]
```diff
@@ -2,12 +2,187 @@
   <div class="content">
     <h2 class="text-center">Selected Monarch Publications</h2>
       <div class="card">
+        <h3>2021</h3>
+          <div class="text">
+            <p>Gene Ontology Consortium. (2021) <b class="title">The Gene Ontology resource: enriching a GOld mine</b> <i>Nucleic Acids Res.</i> <b>49(D1)</b>:D325-D334. [<a target="__blank" href="https://pubmed.ncbi.nlm.nih.gov/33290552/">pubmed</a>]
```
[the remark this answers is not in the document]
Comment (jen-martin) [reply src/views/Publications.md]: done

File: src/views/Publications.md [hunk 2]
```diff
@@ -2,12 +2,187 @@
...
+          </div>
+          <div class="text">
+            <p>Reese JT, Unni D, Callahan TJ, Cappelletti L, Ravanmehr V, Carbon S, Shefchek KA, Good BM, Balhoff JP, Fontana T, Blau H, Matentzoglu N, Harris NL, Munoz-Torres MC, Haendel MA, Robinson PN, Joachimiak MP, Mungall CJ. (2021) <b class="title">KG-COVID-19: A Framework to Produce Customized Knowledge Graphs for COVID-19 Response</b>. <i>Patterns (N Y)</i> <b>2(1)</b>:100155. [<a target="__blank" href="https://pubmed.ncbi.nlm.nih.gov/33196056/">pubmed</a>]
+            </p>
+          </div>
+          <div class="text">
+            <p>Rodríguez de Los Santos M, Rivalan M, David FS, Stumpf A, Pitsch J, Tsortouktzidis D, Velasquez LM, Voigt A, Schilling K, Mattei D, Long M, Vogt G, Knaus A, Fischer-Zirnsak B, Wittler L, Timmermann B, Robinson PN, Horn D, Mundlos S, Kornak U, Becker AJ, Schmitz D, Winter Y, Krawitz PM. (2021) <b class="title">A CRISPR-Cas9-engineered mouse model for GPI-anchor deficiency mirrors human phenotypes and exhibits hippocampal synaptic dysfunctions</b>. <i>Proc Natl Acad Sci U S A.</i> <b>118(2)</b>:e2014481118. [<a target="__blank" href="https://pubmed.ncbi.nlm.nih.gov/33402532/">pubmed</a>]
```
[the remark this answers is not in the document]
Comment (jen-martin) [reply src/views/Publications.md]: done

File: src/views/Publications.md [hunk 3]
```diff
@@ -2,12 +2,187 @@
...
+            </p>
+          </div>
+          <div class="text">
+            <p>Vangay P, Burgin J, Johnston A, Beck KL, Berrios DC, Blumberg K, Canon S, Chain P, Chandonia JM, Christianson D, Costes SV, Damerow J, Duncan WD, Dundore-Arias JP, Fagnan K, Galazka JM, Gibbons SM, Hays D, Hervey J, Hu B, Hurwitz BL, Jaiswal P, Joachimiak MP, Kinkel L, Ladau J, Martin SL, McCue LA, Miller K, Mouncey N, Mungall C, Pafilis E, Reddy TBK, Richardson L, Roux S, Schriml LM, Shaffer JP, Sundaramurthi JC, Thompson LR, Timme RE, Zheng J, Wood-Charlson EM, Eloe-Fadrosh EA. (2021) <b class="title">Microbiome Metadata Standards: Report of the National Microbiome Data Collaborative's Workshop and Follow-On Activities</b>. <i>mSystems.</i> <b>6(1)</b>:e01194-20. [<a target="__blank" href="https://pubmed.ncbi.nlm.nih.gov/33622857/">pubmed</a>]
```
[the remark this answers is not in the document]
Comment (jen-martin) [reply src/views/Publications.md]: done

File: src/views/Publications.md [hunk 4]
```diff
@@ -2,12 +2,187 @@
...
+          </div>
+          <div class="text">
+            <p>Vasilevsky NA, Hosseini M, Teplitzky S, Ilik V, Mohammadi E, Schneider J, Kern B, Colomb J, Edmunds SC, Gutzman K, Himmelstein DS, White M, Smith B, O'Keefe L, Haendel M, Holmes KL. (2021) <b class="title">Is authorship sufficient for today's collaborative research? A call for contributor roles</b>. <i>Account Res.</i> <b>28(1)</b>:23-43. [<a target="__blank" href="https://pubmed.ncbi.nlm.nih.gov/32602379/">pubmed</a>]
+            </p>
+          </div>
+        </div>
+        <div class="card">
+          <h3>2020</h3>
+          <div class="text">
+            <p>Alliance of Genome Resources Consortium. (2020) <b class="title">Alliance of Genome Resources Portal: unified model organism research platform</b>. <i>Nucleic Acids Res.</i> <b>48(D1)</b>:D650-D658. [<a target="__blank" href="https://pubmed.ncbi.nlm.nih.gov/31552413/">pubmed</a>]
```
[the remark this answers is not in the document]
Comment (jen-martin) [reply src/views/Publications.md]: done

File: src/views/Publications.md [hunk 5]
```diff
@@ -2,12 +2,187 @@
...
+            </p>
+          </div>
+          <div class="text">
+            <p>Arnaud E, Laporte MA, Kim S, Aubert C, Leonelli S, Miro B, Cooper L, Jaiswal P, Kruseman G, Shrestha R, Buttigieg PL, Mungall CJ, Pietragalla J, Agbona A, Muliro J, Detras J, Hualla V, Rathore A, Das RR, Dieng I, Bauchet G, Menda N, Pommier C, Shaw F, Lyon D, Mwanzia L, Juarez H, Bonaiuti E, Chiputwa B, Obileye O, Auzoux S, Yeumo ED, Mueller LA, Silverstein K, Lafargue A, Antezana E, Devare M, King B. (2020) <b class="title">The Ontologies Community of Practice: A CGIAR Initiative for Big Data in Agrifood Systems</b>. <i>Patterns (N Y).</i> <b>1(7)</b>:100105. [<a target="__blank" href="https://pubmed.ncbi.nlm.nih.gov/33205138/">pubmed</a>]
```
[the remark this answers is not in the document]
Comment (jen-martin) [reply src/views/Publications.md]: done

File: src/views/Publications.md [hunk 6]
```diff
@@ -2,12 +2,187 @@
...
+            </p>
+          </div>
+          <div class="text">
+            <p>Karlebach G, Hansen P, Veiga DF, Steinhaus R, Danis D, Li S, Anczukow O, Robinson PN. (2020) <b class="title">HBA-DEALS: accurate and simultaneous identification of differential expression and splicing using hierarchical Bayesian analysis</b>. <i>Genome Biol.</i> <b>21(1)</b>:171. [<a target="__blank" href="https://pubmed.ncbi.nlm.nih.gov/32660516/">pubmed</a>]
+            </p>
+          </div>
+          <div class="text">
+            <p>Mabee PM, Balhoff JP, Dahdul WM, Lapp H, Mungall CJ, Vision TJ. (2020) <b class="title">A Logical Model of Homology for Comparative Biology</b>. <i>Syst Biol.</i> <b>69(2)</b>:345-362. [<a target="__blank" href="https://pubmed.ncbi.nlm.nih.gov/31596473/">pubmed</a>]
```
[the remark this answers is not in the document]
Comment (jen-martin) [reply src/views/Publications.md]: done

File: src/views/Publications.md [hunk 7]
```diff
@@ -2,12 +2,187 @@
...
+            </p>
+          </div>
+          <div class="text">
+            <p>Steinhaus R, Gonzalez T, Seelow D, Robinson PN. (2020) <b class="title">Pervasive and CpG-dependent promoter-like characteristics of transcribed enhancers</b>. <i>Nucleic Acids Res.</i> <b>48(10)</b>:5306-5317. [<a target="__blank" href="https://pubmed.ncbi.nlm.nih.gov/32338759/">pubmed</a>]
```
[the remark this answers is not in the document]
Comment (jen-martin) [reply src/views/Publications.md]: done

File: src/views/Publications.md [hunk 8]
```diff
@@ -2,12 +2,187 @@
...
+            </p>
+          </div>
+          <div class="text">
+            <p>Thessen AE, Walls RL, Vogt L, Singer J, Warren R, Buttigieg PL, Balhoff JP, Mungall CJ, McGuinness DL, Stucky BJ, Yoder MJ, Haendel MA. (2020) <b class="title">Transforming the study of organisms: Phenomic data models and knowledge bases</b>. <i>PLoS Comput Biol.</i> <b>16(11)</b>:e1008376. [<a target="__blank" href="https://pubmed.ncbi.nlm.nih.gov/33232313/">pubmed</a>]
+            </p>
+          </div>
+          <div class="text">
+            <p>Touré V, Vercruysse S, Acencio ML, Lovering RC, Orchard S, Bradley G, Casals-Casas C, Chaouiya C, Del-Toro N, Flobak Å, Gaudet P, Hermjakob H, Hoyt CT, Licata L, Lægreid A, Mungall CJ, Niknejad A, Panni S, Perfetto L, Porras P, Pratt D, Saez-Rodriguez J, Thieffry D, Thomas PD, Türei D, Kuiper M. (2020) <b class="title">The Minimum Information about a Molecular Interaction Causal Statement (MI2CAST)</b>. <i>Bioinformatics.</i> btaa622. [<a target="__blank" href="https://pubmed.ncbi.nlm.nih.gov/32637990/">pubmed</a>]
```
[the remark this answers is not in the document]
Comment (jen-martin) [reply src/views/Publications.md]: done

File: src/views/Publications.md [hunk 9]
```diff
@@ -2,12 +2,187 @@
...
+            </p>
+          </div>
+          <div class="text">
+            <p>Wood V, Carbon S, Harris MA, Lock A, Engel SR, Hill DP, Van Auken K, Attrill H, Feuermann M, Gaudet P, Lovering RC, Poux S, Rutherford KM, Mungall CJ. (2020) <b class="title">Term Matrix: a novel Gene Ontology annotation quality control system based on ontology term co-annotation patterns</b>. <i>Open Biol.</i> <b>10(9)</b>:200149. [<a target="__blank" href="https://pubmed.ncbi.nlm.nih.gov/32875947/">pubmed</a>]
```
[the remark this answers is not in the document]
Comment (jen-martin) [reply src/views/Publications.md]: done

File: src/views/Publications.md [hunk 10]
```diff
@@ -2,12 +2,187 @@
...
+            </p>
+          </div>
+          <div class="text">
+            <p>Wood-Charlson EM, Anubhav, Auberry D, Blanco H, Borkum MI, Corilo YE, Davenport KW, Deshpande S, Devarakonda R, Drake M, Duncan WD, Flynn MC, Hays D, Hu B, Huntemann M, Li PE, Lipton M, Lo CC, Millard D, Miller K, Piehowski PD, Purvine S, Reddy TBK, Shakya M, Sundaramurthi JC, Vangay P, Wei Y, Wilson BE, Canon S, Chain PSG, Fagnan K, Martin S, McCue LA, Mungall CJ, Mouncey NJ, Maxon ME, Eloe-Fadrosh EA. (2020) <b class="title">The National Microbiome Data Collaborative: enabling microbiome science</b>. <i>Nat Rev Microbiol.</i> <b>18(6)</b>:313-314. [<a target="__blank" href="https://pubmed.ncbi.nlm.nih.gov/32350400/">pubmed</a>]
```
[the remark this answers is not in the document]
Comment (jen-martin) [reply src/views/Publications.md]: done

File: src/views/Publications.md [hunk 11]
```diff
@@ -2,12 +2,187 @@
...
+            </p>
+          </div>
+          <div class="text">
+            <p>Altinbas L, Bormann N, Lehmann D, Jeuthe S, Wulsten D, Kornak U, Robinson PN, Wildemann B, Kararigas G. (2019) <b class="title">Assessment of Bones Deficient in Fibrillin-1 Microfibrils Reveals Pronounced Sex Differences</b>. <i>Int J Mol Sci.</i> <b>20(23)</b>:6059. [<a target="__blank" href="https://pubmed.ncbi.nlm.nih.gov/31805661">pubmed</a>]
+            </p>
+          </div>
+          <div class="text">
+            <p>Bhushan R, Altinbas L, Jäger M, Zaradzki M, Lehmann D, Timmermann B, Clayton NP, Zhu Y, Kallenbach K, Kararigas G, Robinson PN. (2019) <b class="title">An integrative systems approach identifies novel candidates in Marfan syndrome-related pathophysiology</b>. <i>J Cell Mol Med.</i> <b>23(4)</b>:2526-2535. <a target="__blank" href="https://pubmed.ncbi.nlm.nih.gov/30677223">pubmed</a>]
```
[the remark this answers is not in the document]
Comment (jen-martin) [reply src/views/Publications.md]: done

File: src/views/Publications.md [hunk 12]
```diff
@@ -2,12 +2,187 @@
...
-               [<a href="http://www.ncbi.nlm.nih.gov/pubmed/31127358">pubmed</a>]
+               [<a target="__blank" href="http://www.ncbi.nlm.nih.gov/pubmed/31127358">pubmed</a>]
+            </p>
+          </div>
+          <div class="text">
+            <p>Carbon S, Champieux R, McMurry JA, Winfree L, Wyatt LR, Haendel MA. (2019) <b class="title">An analysis and metric of reusable data licensing practices for biomedical resources</b>. <i>PLoS One</i>. <b>14(3)</b>:e0213090. [<a target="__blank" href="http://www.ncbi.nlm.nih.gov/pubmed/30917137">pubmed</a>]
+            </p>
+          </div>
+          <div class="text">
+            <p>Giglio M, Tauber R, Nadendla S, Munro J, Olley D, Ball S, Mitraka E, Schriml LM, Gaudet P, Hobbs ET, Erill I, Siegele DA, Hu JC, Mungall C, Chibucos MC. (2109) <b class="title">ECO, the Evidence & Conclusion Ontology: community standard for evidence information</b>. <i>Nucleic Acids Res.</i> <b>47(D1)</b>:D1186-D1194. [<a target="__blank" href="http://www.ncbi.nlm.nih.gov/pubmed/30407590">pubmed</a>]
```
[the remark this answers is not in the document]
Comment (jen-martin) [reply src/views/Publications.md]: done

File: src/views/Publications.md [hunk 13]
```diff
@@ -2,12 +2,187 @@
...
+            </p>
+          </div>
+          <div class="text">
+            <p>
+              Gourdine JF, Brush MH, Vasilevsky NA, Shefchek K, Köhler S, Matentzoglu N, Munoz-Torres MC, McMurry JA, Zhang XA, Robinson PN, Haendel MA. (2019) <b class="title">Representing glycophenotypes: semantic unification of glycobiology resources for disease discovery</b>. <i>Database (Oxford). </i> baz114. [<a target="__blank" href="http://www.ncbi.nlm.nih.gov/pubmed/31735951">pubmed</a>]
+            </p>
+          </div>
+          <div class="text">
+            <p>Hansen P, Ali S, Blau H, Danis D, Hecht J, Kornak U, Lupiáñez DG, Mundlos S, Steinhaus R, Robinson PN. (2019) <b class="title">GOPHER: Generator Of Probes for capture Hi-C Experiments at high Resolution</b>. <i>BMC Genomics.</i> <b>20(1)</b>:40. [<a target="__blank" href="http://www.ncbi.nlm.nih.gov/pubmed/30642251">pubmed</a>]
```
[the remark this answers is not in the document]
Comment (jen-martin) [reply src/views/Publications.md]: done

File: src/views/Publications.md [hunk 14]
```diff
@@ -2,12 +2,187 @@
...
+            </p>
+          </div>
+          <div class="text">
+            <p>Hansen P, Gargano M, Hecht J, Ibn-Salem J, Karlebach G, Roehr JT, Robinson PN. (2019) <b class="title">Computational Processing and Quality Control of Hi-C, Capture Hi-C and Capture-C Data</b>. <i>Genes (Basel).</i> <b>10(7)</b>:548. [<a target="__blank" href="http://www.ncbi.nlm.nih.gov/pubmed/31323892">pubmed</a>]
```
[the remark this answers is not in the document]
Comment (jen-martin) [reply src/views/Publications.md]: done

File: src/views/Publications.md [hunk 15]
```diff
@@ -2,12 +2,187 @@
...
... O, Bayat A, Becerra-Solano L, Bentzen HB, Biskup S, Borisov O, Braaten O, Ciaccio C, Coutelier M, Cremer K, Danyel M, Daschkey S, Eden HD, Devriendt K, Wilson S, Douzgou S, Đukić D, Ehmke N, Fauth C, Fischer-Zirnsak B, Fleischer N, Gabriel H, Graul-Neumann L, Gripp KW, Gurovich Y, Gusina A, Haddad N, Hajjir N, Hanani Y, Hertzberg J, Hoertnagel K, Howell J, Ivanovski I, Kaindl A, Kamphans T, Kamphausen S, Karimov C, Kathom H, Keryan A, Knaus A, Köhler S, Kornak U, Lavrov A, Leitheiser M, Lyon GJ, Mangold E, Reina PM, Carrascal AM, Mitter D, Herrador LM, Nadav G, Nöthen M, Orrico A, Ott CE, Park K, Peterlin B, Pölsler L, Raas-Rothschild A, Randolph L, Revencu N, Fagerberg CR, Robinson PN, Rosnev S, Rudnik S, Rudolf G, Schatz U, Schossig A, Schubach M, Shanoon O, Sheridan E, Smirin-Yosef P, Spielmann M, Suk EK, Sznajer Y, Thiel CT, Thiel G, Verloes A, Vrecar I, Wahl D, Weber I, Winter K, Wiśniewska M, Wollnik B, Yeung MW, Zhao M, Zhu N, Zschocke J, Mundlos S, Horn D, Krawitz PM. (2019) <b class="title">PEDIA: prioritization of exome data by image analysis</b> <i>Genet Med.</i> <b>21(12)</b>:2807-2814. [<a target="__blank" href="http://www.ncbi.nlm.nih.gov/pubmed/31164752">pubmed</a>]
```
[the remark this answers is not in the document]
Comment (jen-martin) [reply src/views/Publications.md]: done

File: src/views/Publications.md [hunk 16]
```diff
@@ -2,12 +2,187 @@
...
+            </p>
+          </div>
+          <div class="text">
+            <p>Jackson RC, Balhoff JP, Douglass E, Harris NL, Mungall CJ, Overton JA. (2019) <b class="title">ROBOT: A Tool for Automating Ontology Workflows</b>. <i>BMC Bioinformatics.</i> <b>20(1)</b>:407. [<a target="__blank" href="http://www.ncbi.nlm.nih.gov/pubmed/31357927">pubmed</a>]
+            </p>
+          </div>
+          <div class="text">
+            <p>Jia G, Li Y, Zhang H, Chattopadhyay I, Boeck Jensen A, Blair DR, Davis L, Robinson PN, Dahlén T, Brunak S, Benson M, Edgren G, Cox NJ, Gao X, Rzhetsky A. (2019) <b class="title">Estimating heritability and genetic correlations from large health datasets in the absence of genetic data</b>. <i>Nat Commun.</i> <b>10(1)</b>:5508. [<a target="__blank" href="http://www.ncbi.nlm.nih.gov/pubmed/31796735">pubmed</a>]
```
[the remark this answers is not in the document]
Comment (jen-martin) [reply src/views/Publications.md]: done

File: src/views/Publications.md [hunk 17]
```diff
@@ -16,6 +191,39 @@
...
+            </p>
+          </div>
+          <div class="text">
+            <p>Thomas PD, Hill DP, Mi H, Osumi-Sutherland D, Van Auken K, Carbon S, Balhoff JP, Albou LP, Good B, Gaudet P, Lewis SE, Mungall CJ. (2019) <b class="title">Gene Ontology Causal Activity Modeling (GO-CAM) moves beyond GO annotations to structured descriptions of biological functions and systems</b>. <i>Nat Genet.</i> <b>51(10)</b>:1429-1433. [<a target="__blank" href="http://www.ncbi.nlm.nih.gov/pubmed/31548717">pubmed</a>]
```
[the remark this answers is not in the document]
Comment (jen-martin) [reply src/views/Publications.md]: done

**Review by monicacecilia — APPROVED** [review]
This is wonderful, Jen. Thank you so much. @jen-martin 
:bowtie:  - we can merge now.

**Pull request merged**